account_invoice_pricelist/tests/test_account_move_pricelist.py

```diff
@@ -218,6 +218,20 @@ def setUpClass(cls):
                 ],
             }
         )
+        # Fix currency rate of EUR -> USD to 1.5289
+        cls.usd_currency = cls.env["res.currency"].search([("name", "=", "USD")])
```

Comment:
Why storing this and the next one in `cls`?

Reply:
Not sure I understand well, correct me.
I think we need to set the USD rates to be used in all test_xxx. So, I put it in setUpClass.

Reply:
Yes, but don't store it as class variable. Just use local variables:
```python
usd_currency = cls.env["res.currency"].search([("name", "=", "USD")])
rates = ....
```

Reply:
Oh ok, sorry for that :). Done.